content/blog/2024-09-10-memory-access-amortization.md

```diff
@@ -0,0 +1,158 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Memory Access Amortization in Valkey "
+# `date` is when your post will be published.
+# For the most part, you can leave this as the day you _started_ the post.
+# The maintainers will update this value before publishing
+# The time is generally irrelvant in how Valkey published, so '01:01:01' is a good placeholder
+date= 2024-09-11 01:01:01
+# 'description' is what is shown as a snippet/summary in various contexts.
+# You can make this the first few lines of the post or (better) a hook for readers.
+# Aim for 2 short sentences.
+description= "Improve performance by enabling the processor's speculative execution capability"
```

Comment:
This is the description that shows up on the main valkey page as a description of the blog, so this should be more of a hook for a reader to open it. For 1M RPS we had:

```
Learn about the new performnace improvements in Valkey 8 which reduces cost, improves latency and makes our environment greener.
``` 
(Just realized there is a typo there)